docs/setup/integration.md

```diff
@@ -9,9 +9,9 @@ Integrating the OmiseGO eWallet requires a new setup to be deployed. Feel free t
 |Area of responsibilities|Tasks|
 |------------------------|-----|
 |Token management   | - Create loyalty tokens <br> - Put more loyalty tokens in circulation <br> - Remove loyalty tokens from circulation <br> - Provide user interface for creating new loyalty tokens <br> - Provide user interface for add/remove of loyalty tokens from circulation|
-|Secondary user store|- Create users along with their token balances. The user stored in the Wallet API is solely for identifying and transacting with the user’s token balances.|
+|Secondary user store|- Create users along with their token wallets. The user stored in the Wallet API is solely for identifying and transacting with the user’s token wallets.|
```

Comment:
Do we still need the word "token" before wallets? It's not used anywhere else and still understandable I think.